Report for FP1

cp0153 opened the pull request:
Thank you, 
I just removed those instructions.

cp0153 committed:
added test file and readme

cp0153 committed:
added picture

cp0153 committed:
Update README.md

cp0153 committed:
Update README.md

cp0153 committed:
Update README.md

fgmart-zz commented:
Hi Chris, looks very good, please make the following changes:

remove "Write what you did! Remember that this report must include:" at top of file

remove "How to Prepare and Submit this assignment" and below from bottom of file.

cp0153 committed:
removed instructions at top and bottom of file

remove "Write what you did! Remember that this report must include:" at top of file

remove "How to Prepare and Submit this assignment" and below from bottom of file.

fgmart-zz commented:
Looks good Chris!

(and thank you for knowing that you did not need to issue another pull request :+1:
